concordia/templates/transcriptions/asset_detail.html

```diff
@@ -117,12 +117,12 @@
                     </div>
                 </div>
             </div>
-
-            <div id="asset-image" class="h-100 bg-dark"></div>
+            <div id="asset-image" class="h-100 bg-dark d-print-none"></div>
+            <div class="print-transcription-image d-none d-print-block "><img class="img-fluid" alt="Scanned image of the current content page" src="{% asset_media_url asset %}"></div>
```

Comment:
We. can remove the trailing space:

```suggestion
            <div class="print-transcription-image d-none d-print-block"><img class="img-fluid" alt="Scanned image of the current content page" src="{% asset_media_url asset %}"></div>
```